cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -5573,7 +5575,7 @@ action.cyences_notable_event_action.system_compromised_drilldown = index=* `cs_v
 action.cyences_notable_event_action.attacker_search = | stats sum(count) as count by user
 action.cyences_notable_event_action.attacker_drilldown = index=* `cs_vpn_indexes` tag=authentication action="failure" dest_category="vpn_auth" user=$row.user$
 action.cyences_send_email_action = 1
-action.cyences_notable_event_action.products = VPN
+action.cyences_notable_event_action.products = VPN, Authentication
```

Comment:
same way we need to add the "Email" tag to following alerts:

Email - Calculate UpperBound for Spike In Emails
Email - Hourly Increase In Emails Over Baseline
Email - Daily Spam Email

Include "Vulnerability" tag for "Vulnerability - Detected Vulnerabilities" alert

Also need to check the alerts displayed under "Active Directory & Windows" panel. => may be update product name from cs_filter_savedsearches macro or update products associated to alerts.

Reply:
Also, for following alerts, need to add "Credential Compromise" tag

Credential Compromise - Windows - Credential Dumping through LSASS Access
Credential Compromise - Windows - Credential Dumping via Symlink to Shadow Copy
Credential Compromise - Windows - Credential Dumping via Copy Command from Shadow Copy
Credential Compromise - Windows - Credential Dump From Registry via Reg exe

Reply:
Added Email tag, Vulnerability tag and Network tag to the genertic alerts in that category.
Please let me know if any other generic category is remaining.

Reply:
Windows AD tag fixed.

Reply:
FYI, no need to add Credential Compromise tag as its not a product type or data source type.